helix-core/src/lib.rs

```diff
@@ -91,6 +91,7 @@ pub use syntax::Syntax;
 
 pub use diagnostic::Diagnostic;
 pub use history::History;
+pub use history::StepsOrTimePeriod;
```

Comment:
I think we could just leave this exposed under the `history` module instead of toplevel.

Reply:
Sure, had to make history a public module for this to work.